sessionctx/variable/tidb_vars.go

```diff
@@ -231,6 +231,9 @@ const (
 
 	// RequireSecureTransport indicates the secure mode for data transport
 	RequireSecureTransport = "require_secure_transport"
+
+	// TiFlashFastScan indicates whether use fast scan in tiflash .
+	TiFlashFastScan = "tiflash_fastscan"
```

Comment:
TiFlashEnableFastScan?

Reply:
I think it should keep the same with the session/global variable.